### PR TITLE
Exec codewind-installer to create and validate projects.

### PR DESCRIPTION
**Pull request by hhellyer**
This PR switches to using the installer to generate and validate projects on local disk rather than using the pfe REST API.

Signed-off-by: Howard Hellyer <hhellyer@uk.ibm.com>

### CODE REVIEW

**Comment by tetchel** [comment]
I think we want to fix https://github.com/eclipse/codewind-installer/issues/99 before putting this in though

**Comment by jopit** [comment]
Agree with fixing eclipse/codewind-installer#99 before putting this in, though that issue has been moved to https://github.com/eclipse/codewind/issues/386

**Comment by hhellyer** [comment]
@tetchel @jopit - I've got the fix for eclipse/codewind#386 Once that's merged I'll add the built binaries from jenkins to this PR as well.
I'll delete the lines in the build script that removes the installer from the theia version of the plugin or we won't be able to create projects in che.

File: dev/src/codewind/connection/InstallerWrapper.ts [hunk 1]
```diff
@@ -507,6 +508,61 @@ namespace InstallerWrapper {
...
+
+    export async function createProject(projectPath: string, url: string) : Promise<IInitializationResponse> {
+        return runProjectCommand(projectPath, url);
+    }
+
+    export async function validateProjectDirectory(projectPath: string): Promise<IInitializationResponse> {
+        return runProjectCommand(projectPath, undefined);
+    }
+
+    async function runProjectCommand(projectPath: string, url: string | null | undefined = null ): Promise<IInitializationResponse> {
```
[the remark this answers is not in the document]
Comment (hhellyer) [reply dev/src/codewind/connection/InstallerWrapper.ts]: I think I was switching between Go and Typescript and got my optional parameter syntax totally confused. I've updated it!

**Comment by jopit** [comment]
@hhellyer @elsony Is an external process like the installer actually able to create projects properly when running in Che?

**Comment by jopit** [comment]
We should probably also wait for the fix to https://github.com/eclipse/codewind/issues/389 since there's not point in putting up a version of the vscode extension with regressions.

**Comment by hhellyer** [comment]
@jopit - I've updated the installer binaries.

**Pull request merged**